README.md

```diff
@@ -76,6 +79,15 @@ When file editing is done, you can click button in top right corner of file edit
 
 For more information about `machete` file look at the [reference blog post](https://medium.com/virtuslab/make-your-way-through-the-git-rebase-jungle-with-git-machete-e2ed4dbacd02).
 
+### Discover
+
+Branch layout can be automatically discovered based on your git repository by `Discover Branch Layout` action.
+It detects a layout of 10 recently used branches.
```

Comment:
```suggestion
It detects a layout of around 10 most recently used branches.
```
(`around` since there's a little nuance: 10 most recently used branches are taken into consideration, but at the very end the once that are merged to its parent & have no children are filtered out)

Reply:
(but ofc we don't want to delve into this nuance here)